### ISSUE TITLE
Cannot open PhotonAppSettings on 2022.3.23f1

### ISSUE THREAD

**Issue by tahomasan** [oculus-samples/Unity-Discover#21]
I receive the following error when trying to configure the project per [Project Configuration](https://github.com/oculus-samples/Unity-Discover/blob/main/Documentation/Configuration.md) when trying to open PhotonAppSettings.asset in order paste Photon Voice App ID.

```
ArgumentOutOfRangeException: No method found matching name "GetDrawerTypeForType", signature "Fusion.Editor.UnityInternal+ScriptAttributeUtility+GetDrawerTypeForTypeDelegate", flags "Static, NonPublic" and params: System.Type, candidates are
: Void ResetCachedTypesAndAsset()
UnityEditor.PropertyHandlerCache get_propertyHandlerCache()
Void set_propertyHandlerCache(UnityEditor.PropertyHandlerCache)
Void ClearGlobalCache()
Void PopulateBuiltinAttributes()
Void AddBuiltinAttribute(System.String, System.String, UnityEngine.PropertyAttribute)
System.Collections.Generic.List`1[UnityEngine.PropertyAttribute] GetBuiltinAttributes(UnityEditor.SerializedProperty)
System.Collections.Generic.Dictionary`2[System.Type,UnityEditor.ScriptAttributeUtility+CustomPropertyDrawerContainer[]] BuildDrawerTypeForTypeDictionary()
System.Type GetDrawerTypeForType(System.Type, Boolean)
Int32 CalculateInheritanceLevel(System.Type, Int32, System.Collections.Generic.Dictionary`2[System.Type,System.Int32])
Boolean TryGetDrawerTypeForTypeFromCache(Boolean, Boolean, System.Type, System.Type ByRef)
Boolean TryExtractDrawerTypeForTypeFromCache(Boolean, Boolean, System.Type, System.Type ByRef)
Boolean TryFindDrawers(CustomPropertyDrawerContainer[], CustomPropertyDrawerContainer ByRef)
System.Type GetDrawerTypeForPropertyAndType(UnityEditor.SerializedProperty, System.Type)
System.Collections.Generic.List`1[UnityEngine.PropertyAttribute] GetFieldAttributes(System.Reflection.FieldInfo)
System.Reflection.FieldInfo GetFieldInfoAndStaticTypeFromProperty(UnityEditor.SerializedProperty, System.Type ByRef)
System.Reflection.FieldInfo GetFieldInfoFromProperty(UnityEditor.SerializedProperty, System.Type ByRef)
Boolean GetTypeFromManagedReferenceFullTypeName(System.String, System.Type ByRef)
System.Type GetScriptTypeFromProperty(UnityEditor.SerializedProperty)
System.Reflection.FieldInfo GetFieldInfoFromPropertyPath(System.Type, System.String, System.Type ByRef)
UnityEditor.PropertyHandler GetHandler(UnityEditor.SerializedProperty)
Boolean CanUseSameHandler(UnityEditor.SerializedProperty, UnityEditor.SerializedProperty)
System.Collections.Generic.List`1[System.Reflection.FieldInfo] GetAutoLoadProperties(System.Type)
Int32 <CalculateInheritanceLevel>g__AddOrReplaceInterfaceLevel|20_0(Int32, <>c__DisplayClass20_0 ByRef)
Boolean <TryExtractDrawerTypeForTypeFromCache>g__IsAppropriateDrawerFound|22_0(<>c__DisplayClass22_0 ByRef)
Parameter name: name
Fusion.Editor.ReflectionUtils.GetMethodOrThrow (System.Type type, System.String name, System.Reflection.BindingFlags flags, System.Type delegateType, Fusion.Editor.ReflectionUtils+DelegateSwizzle[] swizzles, Fusion.Editor.ReflectionUtils+DelegateSwizzle& firstMatchingSwizzle) (at Assets/Photon/Fusion/Scripts/Editor/Fusion.Editor.cs:10841)
Fusion.Editor.ReflectionUtils.GetMethodOrThrow (System.Type type, System.String name, System.Reflection.BindingFlags flags, System.Type delegateType) (at Assets/Photon/Fusion/Scripts/Editor/Fusion.Editor.cs:10768)
Fusion.Editor.ReflectionUtils.CreateMethodDelegateInternal (System.Type type, System.String name, System.Reflection.BindingFlags flags, System.Type delegateType) (at Assets/Photon/Fusion/Scripts/Editor/Fusion.Editor.cs:10763)
Fusion.Editor.ReflectionUtils.CreateMethodDelegateInternal[T] (System.Type type, System.String name, System.Reflection.BindingFlags flags) (at Assets/Photon/Fusion/Scripts/Editor/Fusion.Editor.cs:10759)
Fusion.Editor.ReflectionUtils.CreateMethodDelegate[T] (System.Reflection.Assembly assembly, System.String typeName, System.String methodName, System.Reflection.BindingFlags flags) (at Assets/Photon/Fusion/Scripts/Editor/Fusion.Editor.cs:10511)
Rethrow as InvalidOperationException: UnityEditor.CoreModule, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null.UnityEditor.ScriptAttributeUtility.GetDrawerTypeForType with flags: Static, NonPublic and type: Fusion.Editor.UnityInternal+ScriptAttributeUtility+GetDrawerTypeForTypeDelegate
Fusion.Editor.ReflectionUtils.CreateMethodDelegate[T] (System.Reflection.Assembly assembly, System.String typeName, System.String methodName, System.Reflection.BindingFlags flags) (at Assets/Photon/Fusion/Scripts/Editor/Fusion.Editor.cs:10513)
Fusion.Editor.ReflectionUtils.CreateEditorMethodDelegate[T] (System.String editorAssemblyTypeName, System.String methodName, System.Reflection.BindingFlags flags) (at Assets/Photon/Fusion/Scripts/Editor/Fusion.Editor.cs:10483)
Fusion.Editor.UnityInternal+ScriptAttributeUtility..cctor () (at Assets/Photon/Fusion/Scripts/Editor/Fusion.Editor.cs:11651)
Rethrow as TypeInitializationException: The type initializer for 'ScriptAttributeUtility' threw an exception.
Fusion.Editor.PhotonAppSettingsEditor.OnInspectorGUI () (at Assets/Photon/Fusion/Scripts/Editor/Fusion.Editor.cs:8108)
UnityEditor.UIElements.InspectorElement+<>c__DisplayClass72_0.<CreateInspectorElementUsingIMGUI>b__0 () (at <1899c5bd2a6d466081cd705bee01d689>:0)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr, Boolean&)


```

**Comment by gsylvain2** [comment]
Similar issue found on the unity forum, linking here as well: https://forum.unity.com/threads/need-help-part-of-unity-inspector-missing.1575975/

**Comment by gsylvain2** [comment]
If you have access to photon's discord: https://discord.com/channels/87465474098483200/851756040030912532/1227691398351028274

**Comment by gsylvain2** [comment]
@tahomasan The fix is supposed to be available now - quantum v2.1.8.1

**Comment by AlexThiv** [comment]
We integrated a fix in the photon package to address the issue.

https://github.com/oculus-samples/Unity-Discover/commit/2619891b481de365b5474c44232c1b7dccbc2271
